src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -58,9 +141,30 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
                 Padding = new Thickness(1),
                 VerticalAlignment = VerticalAlignment.Center,
             };
-
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = sender as Border;
+            var uiList = CreateDescriptionAsync().Result;
+
+            bool KeepOpen()
+            {
+                var mousePoint = Mouse.GetPosition(border);
+                if (mousePoint.X > border.ActualWidth || mousePoint.X < 0 || mousePoint.Y > border.ActualHeight || mousePoint.Y < 0)
```

Comment:
Is it easier to negate the boolean logic and then you don't have to do the return false/return true?

Reply:
I always do it this way because I think it's easier to read, but I can change it.